pyproject.toml

```diff
@@ -1,61 +1,65 @@
-[project]
+[tool.poetry]
 name = "speXtra"
-version = "0.34-alpha"
-description = "Tool to manage and manipulate  astronomical spectra"
-readme = "README.md"
-requires-python = ">=3.8"
-# When updating the version, also update the versions in .github/workflows/*
-license = {text = "License :: OSI Approved :: MIT License"}
-authors = [
-    {name = "Miguel Verdugo", email="mverduol@gmail.com"},
-]
+version = "0.40.0"
+description = "Tool to manage and manipulate astronomical spectra."
+license = "MIT"
+authors = ["Miguel Verdugo <mverduol@gmail.com>"]
 maintainers = [
-    {name = "Kieran Leschinski", email="kieran.leschinski@unive.ac.at"},
-    {name = "Hugo Buddelmeijer", email="hugo@buddelmeijer.nl"},
+    "Kieran Leschinski <kieran.leschinski@unive.ac.at>",
+    "Hugo Buddelmeijer <hugo@buddelmeijer.nl>",
+    "Fabian Haberhauer <fabian.haberhauer@univie.ac.at>",
 ]
-classifiers=[
-    "Programming Language :: Python :: 3",
-    "License :: OSI Approved :: MIT License",
-    "Operating System :: OS Independent",
+readme = "README.md"
+repository = "https://github.com/AstarVienna/speXtra"
+documentation = "https://spextra.readthedocs.io/en/latest/"
+classifiers = [
     "Intended Audience :: Science/Research",
-    "Topic :: Scientific/Engineering :: Astronomy",
+    "Operating System :: OS Independent",
     "Topic :: Scientific/Engineering :: Astronomy",
 ]
-dependencies = [
-    # Try to keep the dependencies on lower versions that have a wheel
-    # package on PyPI, for minimumdependencies.yml
-    "numpy>=1.18.0",
-    "astropy>=4.0",
-    "synphot>=1.1.0",
-    "pyyaml>5.2",
-    "scipy>=1.4.0",
-]
 
-[project.optional-dependencies]
-test = [
-    "pytest>=5.0.0",
-    "pytest-cov",
-    "matplotlib>=3.2.0",
-]
-docs = [
-    "sphinx>=4.3.0",
-    "sphinx-rtd-theme>=0.5.1",
-    "jupyter_sphinx==0.2.3",
-    "sphinxcontrib-apidoc",
-    "nbsphinx",
-    "numpydoc",
-]
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12"
+numpy = "^1.24.4"
+scipy = "^1.10.0"
+astropy = "^5.2.2"
+synphot = "^1.2.0"
+
+pyyaml = "^6.0.1"
+pooch = "^1.7.0"
+tqdm = "^4.66.1"
+
+astar-utils = "^0.1.3"
```

Comment:
The caret in a 0.x.y version works on the minor version, not the major version. speXtra is therefor not compatible with the latest astar-utils anymore; from pip:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
spextra 0.40.0 requires astar-utils<0.2.0,>=0.1.3, but you have astar-utils 0.2.0b1 which is incompatible.
```

So perhaps we should go back to `>=`  for our 0. dependencies ?

Reply:
I thought this would work, but just checked poetry's docs and apparently it _does_ work as intended for >1.0 versions, but _not_ for <1.0, as seen in your error here. I created #17 , which seems to work. Perhaps not a solutions everywhere, but maybe worth a look.